addons/stock/views/stock_orderpoint_views.xml

```diff
@@ -56,7 +56,7 @@
                 <button name="action_stock_replenishment_info" type="object" icon="fa-info-circle" title="Replenishment Information" invisible="not id"/>
                 <field name="trigger" optional="hide"/>
                 <field name="group_id" optional="hide" groups="stock.group_adv_location"/>
-                <field name="product_min_qty" string="Min" optional="show"/>
+                <field name="product_min_qty" string="Min" optional="show" width="100%"/>
```

Comment:
`100%` seems pretty high :thinking: also are you sure you want % and not px? Also, aren't you supposed to be applying this to the `action_stock_replenishment_info` button?

Reply:
This comment still holds

Reply:
dropped this change